games-engines/scummvm/scummvm-1.9.0.recipe

```diff
@@ -56,41 +58,56 @@ BUILD_REQUIRES="
 	devel:libogg$secondaryArchSuffix
 	devel:libpng$secondaryArchSuffix
 	devel:libSDL$secondaryArchSuffix
+	#devel:libsdl_net$secondaryArchSuffix #not needed yet
 	devel:libtheora$secondaryArchSuffix
 	devel:libvorbis$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
-"
+	"
 
 BUILD_PREREQUIRES="
 	cmd:gcc$secondaryArchSuffix
-	cmd:git
 	cmd:ld$secondaryArchSuffix
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
 	cmd:nasm
 	cmd:pkg_config$secondaryArchSuffix
-"
+	"
 
 BUILD()
 {
 	libtoolize --force --copy --install
 	CPPFLAGS=`freetype-config --cflags` ./configure --prefix=$prefix \
-	--bindir=$appsDir --libdir=$libDir --datarootdir=$dataDir \
-	--mandir=$manDir --docdir=$docDir \
-	--disable-debug --enable-release #--disable-all-engines
-	#--disable-all-engines can be used to only compile scummvm without
-	#engines, which makes it build faster, those that are still under
-	#construction can be built with --enable-all-engines for testing purposes
-
+		--bindir=$appsDir --libdir=$libDir --datarootdir=$dataDir \
+		--mandir=$manDir --docdir=$docDir \
+		--disable-debug --enable-release --disable-eventrecorder \
+		#--disable-all-engines #can be used to compile scummvm without \
+		#engines, which makes it build faster (to test), those that \
+		#are still under construction can be built with --enable-all-engines \
+		#for testing purposes
 	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
 	mv $appsDir/scummvm $appsDir/ScummVM
-	addResourcesToBinaries $portDir/additional-files/scummvm.rdef \
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		$portDir/additional-files/scummvm.rdef > scummvm.rdef
```

Comment:
The second grep I was interested in was:
```
git remote -v | grep Begasus/haikuports
```

Your "origin" alias should point to **_your_** copy, not to the main repo.
To fix this you should:
```
git remote set-url origin https://github.com/Begasus/haikuporter.git
```
After that,
```
git remote -v | grep Begasus/haikuports
```
should yeld:
```
origin https://github.com/Begasus/haikuports.git (fetch)
origin https://github.com/Begasus/haikuports.git (push)
```

You will also need to:
```
git remote add upstream https://github.com/haikuports/haikuporter.git
```
in order to pull what other contributors push to the main repo.
And to refresh your local master:
```
git checkout master
git pull upstream master
```
Then, to push your local master to your remote clone:
```
git checkout master
git push
```
(of course, if you are already on your master branch, you can skip the "git checkout master".)